src/nix/repl.cc

```diff
@@ -396,15 +396,16 @@ StorePath NixRepl::getDerivationPath(Value & v) {
 
 bool NixRepl::processLine(std::string line)
 {
-    if (line == "") return true;
+    size_t s = line.find_first_not_of(" \n\r\t");
+    if (s == std::string::npos) return true;
```

Comment:
Maybe just do `line = trim(line)`?

Reply:
I thought about this initially but I don't think the extra copy is necessary because both `arg` and `command` already remove white spaces and `line` only serves as an intermediate object where they are extracted from (please correct me if I am wrong as I am not very well-versed in C++).

Reply:
No need to worry about efficiency here. The microsecond it takes to call `trim` isn't going to be noticeable in an interactive function :-)

Reply:
I see, I have applied your suggestion.